libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/datasource/PartitionStat.java

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.openhouse.datalayout.datasource;
+
+import java.io.Serializable;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/** Represents table partition. */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class PartitionStat implements Serializable {
```

Comment:
Meta question given you made it Serializable where are we persisting TablePartitionStat. Are we persisting it for all files in a table? If so that can be a long list.

Reply:
Removed serializable b/c custom serde is used